topics/admin/tutorials/job-destinations/tutorial.md

```diff
@@ -722,10 +667,20 @@ The cores parameter can be verified from the output of the tool. The walltime ca
 > ```
 {: .code-out}
 
-{% snippet topics/admin/faqs/git-commit.md page=page %}
-
 {% snippet topics/admin/faqs/missed-something.md step=9 %}
 
+
+# More on TPV
+
+The goal of this tutorial is to provide a quick overview of some of the basic capabilities of TPV. However, there are numerous features that we have not convered such as:
+a. Custom code blocks - execute arbitrary python code blocks, access additional context variables etc.
+a. User and Role Handling - Add scheduling constraints based on the user's email or role
+b. Metascheduling support - Perform advanced querying and filtering prior to choosing an appropriate destination
+c. Job resubmissions - Resubmit a job if it fails for some reason
+d. Linting, formatting and dry-run - Automatically format tpv rule files, catch potential syntax errors and perform a dry-run to check where a tool would get scheduled.
```

Comment:
I worry about the lack of linting, should we include this somehow? It might reduce user error especially for 'complex' new syntax like python in yaml. Should we include something like https://github.com/usegalaxy-eu/ansible-tpv-lint as part of the playbook? Maybe that would be 'easy'? @mira-miracoli any thoughts on this?

Reply:
The ansible tpv lint would certainly be a nice thing to incorporate.